mdlp/_mdlp.pyx

```diff
@@ -113,7 +120,7 @@ cdef bint reject_split(np.ndarray[np.int64_t, ndim=1] y, int start, int end, int
     return gain <= 1 / N * (log(N - 1) + delta)
 
 @cython.boundscheck(False)
-def find_cut(np.ndarray[np.int64_t, ndim=1] y, int start, int end):
+def find_cut(np.ndarray[np.float64_t, ndim=1] col, np.ndarray[np.int64_t, ndim=1] y, dict labels_map, int start, int end):
     """Finds the best cut between the specified interval.
 
     If k split the array into two sub arrays A and B, then
```

Comment:
I need to freshen up this documentation when I have the time. "Best cut" is too vague. The definition is really taken from the original paper. (MDLPC Criteron.)